device_backends/LogicalNameMapping/src/LNMMathPlugin.cc

```diff
@@ -80,9 +80,6 @@ namespace ChimeraTK { namespace LNMBackend {
     // check if this is the first call to openHook: perform tasks which could not be done in the constructor because
     // the backend is not yet available there.
     if(_backend._empty()) {
-      // make sure the register info is up to date
```

Comment:
Do you understand why this was here? What was it's intention, and why is it working without? And what did it do to disturb the wanted behaviour?